app/lib/fullUrlUtils.js

```diff
@@ -0,0 +1,15 @@
+function url(req) {
```

Comment:
This module doesn't need to be complex as this. I reckon you could just have the following:
```js
module.exports = req => `https://${req.hostname}${req.originalUrl}`;
}
```

I'd also rename it to be `completeOriginalUrl`.

Reply:
Done